### ISSUE TITLE
Multimodal predictor reaches max_epochs = 10 and stops

### ISSUE THREAD

**Issue by kgene521** [autogluon/autogluon#3959]
I'm working with the Multimodal predictor classifying text into multiple classes. It works until it reaches epoch = 10 and then stops. Below is the extract of the logging info:

====================================================================
Global seed set to 0
Using 16bit Automatic Mixed Precision (AMP)
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
[rank: 0] Global seed set to 0
Initializing distributed: GLOBAL_RANK: 0, MEMBER: 1/2
[rank: 1] Global seed set to 0
Initializing distributed: GLOBAL_RANK: 1, MEMBER: 2/2
----------------------------------------------------------------------------------------------------
distributed_backend=nccl
All distributed processes registered. Starting with 2 processes
----------------------------------------------------------------------------------------------------

You are using a CUDA device ('NVIDIA RTX A5000') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0,1]
LOCAL_RANK: 1 - CUDA_VISIBLE_DEVICES: [0,1]

  | Name              | Type                         | Params
-------------------------------------------------------------------
0 | model             | HFAutoModelForTextPrediction | 183 M 
1 | validation_metric | MulticlassAccuracy           | 0     
2 | loss_func         | CrossEntropyLoss             | 0     
-------------------------------------------------------------------
183 M     Trainable params
0         Non-trainable params
183 M     Total params
735.385   Total estimated model params size (MB)
[W reducer.cpp:1300] Warning: find_unused_parameters=True was specified in DDP constructor, but did not find any unused parameters in the forward pass. This flag results in an extra traversal of the autograd graph every iteration,  which can adversely affect performance. If your model indeed never has any unused parameters in the forward pass, consider turning this flag off. Note that this warning may be a false positive if your model has flow control causing later iterations to have unused parameters. (function operator())
[W reducer.cpp:1300] Warning: find_unused_parameters=True was specified in DDP constructor, but did not find any unused parameters in the forward pass. This flag results in an extra traversal of the autograd graph every iteration,  which can adversely affect performance. If your model indeed never has any unused parameters in the forward pass, consider turning this flag off. Note that this warning may be a false positive if your model has flow control causing later iterations to have unused parameters. (function operator())
Epoch 0, global step 187: 'val_acc' reached 0.66520 (best 0.66520), saving model to '/mnt/dataone/data/models/traincc_no_numbers_mm_rows_50599_test_size_0.1_2024-03-02_20-37-17/epoch=0-step=187.ckpt' as top 3
Epoch 0, global step 376: 'val_acc' reached 0.74680 (best 0.74680), saving model to '/mnt/dataone/data/models/traincc_no_numbers_mm_rows_50599_test_size_0.1_2024-03-02_20-37-17/epoch=0-step=376.ckpt' as top 3
Epoch 1, global step 563: 'val_acc' reached 0.78680 (best 0.78680), saving model to '/mnt/dataone/data/models/traincc_no_numbers_mm_rows_50599_test_size_0.1_2024-03-02_20-37-17/epoch=1-step=563.ckpt' as top 3
Epoch 1, global step 752: 'val_acc' reached 0.83040 (best 0.83040), saving model to '/mnt/dataone/data/models/traincc_no_numbers_mm_rows_50599_test_size_0.1_2024-03-02_20-37-17/epoch=1-step=752.ckpt' as top 3
----------
---- removed for clarity 
----------
Epoch 8, global step 3195: 'val_acc' reached 0.91240 (best 0.91240), saving model to '/mnt/dataone/data/models/traincc_no_numbers_mm_rows_50599_test_size_0.1_2024-03-02_20-37-17/epoch=8-step=3195.ckpt' as top 3
Epoch 8, global step 3384: 'val_acc' reached 0.91560 (best 0.91560), saving model to '/mnt/dataone/data/models/traincc_no_numbers_mm_rows_50599_test_size_0.1_2024-03-02_20-37-17/epoch=8-step=3384.ckpt' as top 3
Epoch 9, global step 3571: 'val_acc' reached 0.91760 (best 0.91760), saving model to '/mnt/dataone/data/models/traincc_no_numbers_mm_rows_50599_test_size_0.1_2024-03-02_20-37-17/epoch=9-step=3571.ckpt' as top 3
Epoch 9, global step 3760: 'val_acc' reached 0.91800 (best 0.91800), saving model to '/mnt/dataone/data/models/traincc_no_numbers_mm_rows_50599_test_size_0.1_2024-03-02_20-37-17/epoch=9-step=3760.ckpt' as top 3

`Trainer.fit` stopped: `max_epochs=10` reached.
====================================================================

My question is about the last statement: `Trainer.fit` stopped: `max_epochs=10` reached. In my case it seems that the accuracy can still be improved has the number of epochs was increased. 

I couldn't find any documentation where it's possible to increase the number of epochs in either Multimodal Predictor fit() or its constructor. 

Thank you

**Comment by kgene521** [comment]
I found the yaml files controlling this and other parameters. The location of these files are:

.virtualenvs/<env_name>/lib/pythonN.MM/site-packages/autogluon/multimodal/configs

the optimization subdirectory contains the default.yaml file with the max_epochs parameter. Other directories are model, environment, data, etc.

Hope this helps someone else searching for these config files.
